assets/app/components/site/Pages/navigationJsonPageList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PageListItem from './pageListItem';
+
+const navigationJsonContent = ({ site }) => {
```

Comment:
This is probably nitpicky, and maybe only an issue that I have, but I really like the render function to be the last one in component.

It makes it more explicit that this component is essentially a function that returns some DOM.

Do other people have thoughts on this?